### PR TITLE
Update MadhuSaini22 Profile

### PR DESCRIPTION
**Pull request by MadhuSaini22**
## Fixes Issue
Refactored MadhuSaini22 profile

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4314"></a>

### CODE REVIEW

**Review by Krishnapro — APPROVED** [review]
Looks good to merge

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

File: data/MadhuSaini22.json [hunk 1]
```diff
@@ -30,35 +41,52 @@
...
+      "date": "2022",
+      "icon": "SiOpensourceinitiative",
+      "color": "#000",
+      "description": "My open source journey started after joining EddieHub and 4C community",
+      "url": "https://github.com/MadhuSaini22"
+    },
+     {
+      "title": "Became Staff Member at 4C",
+      "date": "2022",
+      "icon": "CgCommunity",
```
Comment (kumarsonsoff3) [inline data/MadhuSaini22.json:86]: `CgCommunity` icon is not supported by LinkFree, the default globe icon will be placed here in this case. If you want, you can update it to a [supported icon](https://linkfree.eddiehub.io/icons).

**Review by kumarsonsoff3 — COMMENTED** [review]
Rest looks good to me! 🎉

**Comment by MadhuSaini22** [comment]
> Rest looks good to me! 🎉

just updated :) THanks for pointing out!

**Review by kumarsonsoff3 — APPROVED** [review]

**Pull request merged**